src/common/components/LineChart/LineChart.module.css

```diff
@@ -0,0 +1,21 @@
+.overlay {
+  fill: none;
+  pointer-events: all;
+}
+
+.focus circle {
+  fill: #f1f3f3;
+  stroke: #6f257f;
+  stroke-width: 5px;
+}
+
+.hover-line {
+  stroke: #6f257f;
+  stroke-width: 2px;
+  stroke-dasharray: 2, 4;
+}
+
+.tooltip {
```

Comment:
I can find only usages of `.tooltip` class, seems like other classes are not used.
Please point where are they used or remove them if non needed

Reply:
fixed